src/tools/pub/dependencies.md

```diff
@@ -139,7 +139,7 @@ stored in a [Git][] repository.
 {% prettify yaml tag=pre+code %}
 dependencies:
   kittens:
-    git: git://github.com/munificent/kittens.git
+    git: git@github.com/munificent/kittens.git
```

Comment:
For github and similar the format is either:
 * `git@github.com:munificent/kittens.git` (this uses SSH, and requires you to have an SSH key registered with github)
 * `https://github.com/munificent/kittens.git` (provides anonymous read-only access)

**You are missing the colon** in `git@github.com:munificent/kittens.git`, which is a slash in HTTPS.

In general, `pub get` will shell out to `git` so any URL that works with `git clone <url>` should work.

Commonly the useful advice would be to use:
 * `git@github.com:munificent/kittens.git` for private repositories, and,
 * `https://github.com/munificent/kittens.git` for public repositories.

Reply:
Fixed.